### PR TITLE
Preserving the Executable path field in debug properties page.

### LINKED ISSUE #656: Crash when clicking "Browse" in debug properties page if path is not valid
- Create a new .NET Core console app
- Go to project properties\Debug
- Change Launch dropdown to Executable, set Executable to the path to dotnet.exe
- Set Executable to something with an invalid path (ie with quote characters)
- Click "Browse"

Expected: Browse dialog is shown
Actual: VS crashes

Since the Windows Explorer "Copy Path" command includes quotes around the copied path, it's pretty easy to run into this.  It might be a good idea to strip off quotes around paths in general.

### LINKED ISSUE #657: Executable path field not preserved in debug properties page
- Create a new .NET Standard Class Library
- Go to project properties\Debug
- Set Launch to Executable, and put the path to an `Exe` in the Executable field
- Save all
- Close and reopen project properties\Debug

EXPECTED: Executable field value is preserved
ACTUAL: Executable field is blank

The path seems to be written to the launchSettings.json file, so it seems like it's not loading correctly from that file to display on the property page.

@jinujoseph

### PR DESCRIPTION
**Pull request by jinujoseph**
fixes #657   & fixes #656 

IISExpress is not supported so taking that out of the condition too.
@dsplaisted

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs [hunk 1]
```diff
@@ -118,12 +118,14 @@ public LaunchType SelectedLaunchType
...
                         {
-                            ExecutablePath = String.Empty;
-                        }
-                        else if (_selectedLaunchType.CommandName == ProfileCommandNames.IISExpress)
-                        {
-                            ExecutablePath = String.Empty;
-                            HasLaunchOption = true;
+                            if (SelectedDebugProfile != null && !String.IsNullOrWhiteSpace(SelectedDebugProfile.ExecutablePath))
+                            {
+                                ExecutablePath = SelectedDebugProfile.ExecutablePath;
```
Comment (dsplaisted) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs]: The setter for `ExecutablePath` will just store the value back to `SelectedDebugProfile.ExecutablePath`, so this line shouldn't be needed.  I think a simpler way to write the code would be:

```C#
if (ExecutablePath == null)
{
    ExecutablePath = String.Empty;
}
```

Comment (jinujoseph) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs]: @dsplaisted , you are right , fixed it.

**Comment by srivatsn** [comment]
@dotnet-bot test this please

**Comment by jinujoseph** [comment]
@dsplaisted , i started off with that approach then was thinking , i will have to strip all special characters. I am checking this in, let me know if you feel strongly about it. I will create another PR with it.

**Pull request merged**